test/fixtures/PublishCommand/anonymous/packages/anonoymous/package.json

```diff
@@ -0,0 +1,8 @@
+{
+  "dependencies": {
```

Comment:
I actually don't know why the commits were so messed up, I fixed most issues, reverted the merge and sent the new commits.